packages/okta-react-native/index.js

```diff
@@ -28,6 +33,14 @@ export const createConfig = async({
   assertRedirectUri(redirectUri);
   assertRedirectUri(endSessionRedirectUri);
 
+  const { origin } = Url(discoveryUri);
+  authClient = new OktaAuth({ 
+    issuer: origin,
```

Comment:
Using the origin as issuer will be using the ORG level authentication server. Is this what we want?

Reply:
As it only makes AuthN API request,  not sure is the origin/baseUrl can work properly here? Or we need to add one more field in the config? Any suggestions?

Reply:
Users can configure a custom authorization server it would look like ORIGIN/oauth2/AUTH_SERVER_ID

What is the "discoveryUri"? Is it not the issuer (auth server)?

Reply:
It's for the `.well_known` endpoint. The url may come with `/.well_know/xxx`. Probably expose a new field can be easier.